pom.xml

```diff
@@ -39,6 +39,8 @@
         <module>test</module>
         <module>tracing-opentelemetry</module>
         <module>spring-boot3-dummy</module>
+        <module>hibernate-6-integrationtests</module>
+        <module>spring-boot-3-integrationtests</module>
```

Comment:
That's fine for now. We should move those tests to the integration tests some other time/commit.